### ISSUE TITLE
Changes between version 0.1.2 and 0.1.3

### ISSUE THREAD

**Issue by omstuhler** [vanatteveldt/rsyntax#22]
Is there a summary somewhere of what changed between versions 0.1.2 and 0.1.3? I have some script that was written based on 0.1.2 and I'd like to make sure it works properly with the new version of the package.

**Comment by kasperwelbers** [comment]
Hi @omstuhler,

We try to stick to the major.minor.patch versioning, so any changes in the patch region should not break scripts. In this specific update we addressed an edge case bug. If we do a minor update we'll include a list of (unbreaking) changes. 

In general, due to how R works, we'll try to avoid making any breaking changes at all. If at some point we think we need to change a big thing like the query syntax, we're more likely to add it as a separate feature. If breaking changes are unpreventable, we'll provide deprecation/change warnings.

TLDR; your script should still work, and if it doesn't we messed up somehow. If there is a change in results, this should be because we fixed something rather than broke it.

**Comment by omstuhler** [comment]
Hi Kasper,

Thank you for the clarification, I really appreciate it! Just to be clear, I'm not saying I have any indication that something stopped working. Just wanted to make sure.

**Comment by kasperwelbers** [comment]
Good to hear! And if anything somehow does not work as expected please do let me know.
